explorer/explorerroutes.go

```diff
@@ -1796,6 +1835,13 @@ func (exp *explorerUI) AgendasPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	for i, a := range agenda {
```

Comment:
Is this just to get the number to display at 2 decimal places? 

`{{printf "%.2f" .QuorumProgress}}` in the template is probably cleaner.